descriptors/ModuleDescriptor-template.json

```diff
@@ -88,7 +88,7 @@
     },
     {
       "id": "instance-storage",
-      "version": "7.4",
+      "version": "7.5",
```

Comment:
Why has the interface version changed? The permissions are part of the implementation, as far as I can tell, the interface itself has not changed.

Reply:
Make sense, I'll revert this change.